WooCommerce/WooCommerceTests/Notifications/LocalNotificationTests.swift

```diff
@@ -3,56 +3,14 @@ import XCTest
 
 final class LocalNotificationTests: XCTestCase {
 
-    func test_storeCreationComplete_scenario_returns_correct_notification_contents() throws {
+    func test_unknown_scenario_returns_correct_notification_contents() throws {
```

Comment:
Nit: We should not need test for the placeholder case.

Reply:
Same comment for the `PushNotificationsManagerTests` file.

Reply:
Thanks! Handled in https://github.com/woocommerce/woocommerce-ios/pull/12443/commits/0ee95d1a98a24a83f6e5dcd5c92fe4ba5e09f90e